### PR TITLE
Work around trivy rate limit errors

### PR DESCRIPTION
**Pull request by Shubham-astro**
## Details

This PR intends to Use Trivy's GitHub Token

## Related Issues

Related https://github.com/astronomer/issues/issues/6689

### CODE REVIEW

File: git-sync/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-3.6.9
+3.6.9-1
```
Comment (pgvishnuram) [inline git-sync/version.txt:1]: why is this version being updated is this to check the pipeline

Comment (danielhoherd) [reply git-sync/version.txt:1]: @pgvishnuram this was so we could see a build happen, which is needed to trigger the trivy rate limit bug we are investigating.

File: .circleci/continue-config.yml.j2 [hunk 2]
```diff
@@ -43,6 +43,10 @@ workflows:
           directory: {{ directory }}
           requires:
             - test
+          context:
+            - github-repo
+            - trivy-db
+
```
Comment (danielhoherd) [inline .circleci/continue-config.yml.j2:49]:
```suggestion
```

Comment (Shubham-astro) [reply .circleci/continue-config.yml.j2]: done

File: git-sync/version.txt [hunk 3]
```diff
@@ -1 +1 @@
-3.6.9
+3.6.9-4
```
Comment (pgvishnuram) [inline git-sync/version.txt:1]: can you remove these

Comment (Shubham-astro) [reply git-sync/version.txt]: done

**Review by pgvishnuram — APPROVED** [review]

**Review by danielhoherd — APPROVED** [review]
Looks great.

**Pull request merged**